### PR TITLE
hsetroot: new pkg at 1.0.5

### PR DESCRIPTION
**Pull request by sirtomato999**
Read: <getkiss.org/guidestones.txt>

## Description of package

a simple wallpaper changer that supports solid colors, gradients, and images

## New package
- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/hsetroot/build [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+make DESTDIR="$(realpath "$1")" PREFIX=/usr install
```
Comment (dylanaraps) [inline community/hsetroot/build]: Why the `realpath`?

File: community/hsetroot/depends [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pkgconf
```
Comment (dylanaraps) [inline community/hsetroot/depends]: Remove this.

File: community/hsetroot/sources [hunk 3]
```diff
@@ -0,0 +1 @@
+git+https://github.com/himdel/hsetroot/
```
Comment (dylanaraps) [inline community/hsetroot/sources]: Why git?

File: community/hsetroot/depends (hunk 2 above)
Comment (sirtomato999) [reply community/hsetroot/depends]: Sorry, my bad

File: community/hsetroot/sources (hunk 3 above)
Comment (sirtomato999) [reply community/hsetroot/sources]: It's a git repo, what do you mean

File: community/hsetroot/build (hunk 1 above)
Comment (sirtomato999) [reply community/hsetroot/build]: libtool complains about relative paths

File: community/hsetroot/sources (hunk 3 above)
Comment (dylanaraps) [reply community/hsetroot/sources]: Why not a release tarball? https://github.com/himdel/hsetroot/releases

File: community/hsetroot/build (hunk 1 above)
Comment (dylanaraps) [reply community/hsetroot/build]: `$1` isn't relative.

Comment (sirtomato999) [reply community/hsetroot/build]: oh, sorry, I was manually running build with a relative path.

File: community/hsetroot/sources (hunk 3 above)
Comment (sirtomato999) [reply community/hsetroot/sources]: 👍 ok

**Comment by dylanaraps** [comment]
Build should be:

```sh
#!/bin/sh -e

mkdir -p "$1/usr/bin"

make
make PREFIX="$1/usr" install
```

There is no `DESTDIR` support and `make install` doesn't create the directories.

**Comment by sirtomato999** [comment]
oops

**Comment by sirtomato999** [comment]
Sorry, I feel really bad about doing all this stuff wrong 😄

**Pull request merged**